FILES.deploy-tools/usr/local/lib/ndslabs/ansible/roles/cluster-backup/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+backup_image_name: davidraila/cluster-backup:NDS-348
```

Comment:
cluster-backup repository needs be transferred to nds-org, and an auto-build needs to be set up for ndslabs - I can set that part up once ownership has been transferred, but you'll need to update backup_image_name to point to *ndslabs/cluster-backup:latest*